src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler.cs

```diff
@@ -35,6 +40,51 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.AutomaticCompletion
     [Order(After = PredefinedCompletionNames.CompletionCommandHandler)]
     internal class AutomaticLineEnderCommandHandler : AbstractAutomaticLineEnderCommandHandler
     {
+        private static readonly string s_bracePair = string.Concat(
+            SyntaxFacts.GetText(SyntaxKind.OpenBraceToken),
+            Environment.NewLine,
```

Comment:
i'm always concerned about hardcoded Env.NewLine.  I would prefer we respect the actual buffer newline, which should be available.